src/Translator.php

```diff
@@ -102,6 +102,15 @@ private function checkEnv(): void
         } catch (ProcessFailedException $e) {
             throw new RuntimeException('Could not find wget command. Please install it.');
         }
+        $txToken = getenv('TX_TOKEN');
+        if (!$txToken) {
+            $message = implode(' ', [
+                'Could not get a valid token from TX_TOKEN environment variable.',
+                'Generate this at https://app.transifex.com/user/settings/api/ and add it to `~/.transifexrc` file by' .
+                ' manually running `tx push -s` on a supported repository and entering your token when prompted.'
+            ]);
+            throw new RuntimeException($message);
+        }
```

Comment:
This looks like we're checking if an environment variable is present..... but not using it? Is this actually meant to check that `~/.transifexrc` is set correctly? If so, check that instead.

Reply:
Seems like you actually need to pass it in.  If you manually run `tx push -s` then `~/.transifexrc` will be read, however in a PHP context there's the `~` directory presumably doesn't work.  The tx binary (not our PHP program) will read `TX_TOKEN` so that it doesn't need to read from `~/.transifexrc`